cli/command/stack/deploy.go

```diff
@@ -520,6 +521,14 @@ func convertService(
 	return serviceSpec, nil
 }
 
+func convertExtraHosts(extraHosts map[string]string) []string {
+	hosts := []string{}
+	for host, ip := range extraHosts {
+		hosts = append(hosts, fmt.Sprintf("%s %s", host, ip))
```

Comment:
I think it is, see https://github.com/docker/docker/pull/28031/files#diff-728a7fe135201867a6b4b645adc9553dR405